recipes/pantax/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    cp -rf $SRC_DIR $PREFIX
+else
+    cd ${SRC_DIR}/gurobi11
+    ${PYTHON} -m pip install gurobipy-11.0.2-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl
```

Comment:
I think it is not allowed to install dependencies with Pip like this.

Reply:
Where does it install it, by the way ? It looks like it is no inside `$PREFIX`

Reply:
Why? Are there any rules that restrict this approach? I have already installed the artifact locally and tested it sucessfully, and PanTax is works well. Gurobipy, as a Python package, should be installed in the `/path/to/env/lib/python3.10/site-packages/gurobipy`. `${PREFIX}` should be `/path/to/env/bin`，so it will certainly not be installed there.

Reply:
About the question of `vg` version. I use `vg` version 1.52 to ensure the robustness and reproducibility of the our tool. All my experiments are run on this version. In addition, I have a dependency `PGGB`. Pggb depends on `vg` version 1.40. Therefore, I cannot directly add `vg` to the dependency, otherwise the `vg` version 1.40 will be overwritten by the new version of `vg`.